README.md

```diff
@@ -1,9 +1,5 @@
 # curriculum-databases-projects-template
 
```

Comment:
- [ ] Why don't you use the correct project name ? , Kindly  use the name that gives an idea of what you did 👍 , like `vet_clinic db` 😄 😄 😄